evergreen/config.yml

```diff
@@ -1963,6 +1980,21 @@ buildvariants:
   - name: compile_test_coverage
   - name: finalize_coverage_data
 
+- name: ubuntu-trace-logging
+  display_name: "Ubuntu (Trace Logging Enabled)"
+  run_on: ubuntu2204-arm64-large
+  allowed_requesters: [ "patch", "ad_hoc" ]
```

Comment:
For now this runs in the nightly build. Bad behavior in trace-level logging is something we should catch but isn't something I think we need to try to catch on every PR patch build.